builder/bundles.go

```diff
@@ -312,9 +312,27 @@ func repoPkgFromNoopInstall(installOut string) (repoPkgMap, repoRpmMap) {
 	return repoPkgs, repoRpmMap
 }
 
+func queryRpmName(packageCmd []string, pkgName string) string {
+	queryStringRpm := merge(
+		packageCmd,
+		"repoquery",
+		"--location",
+		"--repo local",
```

Comment:
I don't think this query is returning the rpm file names from JUST the **local** repo. If the package exists in both the `local` and `clear` repo, both the rpm file names are returned.
Can you please confirm this?

Reply:
fixed it!